plugins/search/src/components/SearchResult/SearchResult.tsx

```diff
@@ -159,8 +166,19 @@ export const SearchResult = ({ searchQuery }: SearchResultProps) => {
       setFilteredResults(withFilters);
     }
   }, [filters, searchQuery, results]);
-
-  if (loading || error || !results) return null;
+  if (loading) {
+    return <Progress />;
+  }
+  if (error) {
+    return (
+      <Alert severity="error">
+        Error encountered while fetching search results. {error.toString()}
+      </Alert>
+    );
+  }
+  if (!results || results.length === 0) {
+    return <EmptyState missing="info" title="Sorry, no results were found" />;
```

Comment:
@emmaindal, I amended changes, so the history changed. But, here is fix

Reply:
Great! I added a commit suggestion of just changing the type of the empty state component below

Reply:
sorry, i missed your second comment. Just accepted your suggestion